### PR TITLE
🚚 Reorganize repo

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
Reorganize repository to make "space" for a C client.

## Related issue/s
https://github.com/ITISFoundation/osparc-simcore/issues/4345

## Checklist

- [ ] Update [``api/openapi.json``](./api/openapi.json)
- [ ] Update code
- [ ] Update doc
- [ ] Follow [NOTES.md](./NOTES.md)

### CODE REVIEW

**Review by pcrespov — CHANGES_REQUESTED** [review]
This layout will break the Configuration of *Pages*. That service expects a `/doc/index.html`

see https://github.com/ITISFoundation/osparc-simcore-clients/settings/pages

**Comment by bisgaard-itis** [comment]
> This layout will break the Configuration of _Pages_. That service expects a `/doc/index.html`
> 
> see https://github.com/ITISFoundation/osparc-simcore-clients/settings/pages

Thanks. Will fix

**Comment by bisgaard-itis** [comment]
@pcrespov this one should be ready now. It doesn't introduce any new functionality. It only moves stuff around. I checked the `make` commands work and make sure to "route" the website correctly (that's the only actual change to the functionality of this repo). Next I will move the codegeneration scripts/commands here and make sure the documentation is handled with as little manual work as possible

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,9 +1 @@
...
-
-black
-bump2version
-isort
-notedown
-nox
-pre-commit
-pylint
\ No newline at end of file
+-r clients/python/requirements-dev.txt
```
Comment (pcrespov) [inline requirements-dev.txt:1]: why do you need thisone here?

File: scripts/common.Makefile [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Globals to be included into all Makefiles
+# specification of the used openapi-generator-cli (see also https://github.com/ITISFoundation/openapi-generator)
+
+REPO_ROOT               := $(realpath $(dir $(abspath $(lastword $(MAKEFILE_LIST))))/..)
```
Comment (pcrespov) [inline scripts/common.Makefile:4]: NOTE that ``$(CURDIR)`` is part of `Makefile` for current's file directory! https://www.gnu.org/software/make/manual/html_node/Quick-Reference.html

**Review by pcrespov — APPROVED** [review]

File: scripts/common.Makefile (hunk 2 above)
Comment (bisgaard-itis) [reply scripts/common.Makefile:6]: Right, but I seem to encounter issues when this one is included into other make files. Basically when I use `$(CURDIR)` here I get a different `REPO_ROOT` in the root Makefile and the clients/python Makefile. That's why I did it like this.

File: requirements-dev.txt (hunk 1 above)
Comment (bisgaard-itis) [reply requirements-dev.txt:1]: Good question. Maybe I don't. I will double check and remove if not.

File: scripts/common.Makefile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+GIT_USER_ID := ITISFoundation
+GIT_CLIENT_REPO_ID := osparc-simcore-clients
+GIT_OPENAPI_REPO_ID := openapi-generator
+GENERATOR_NAME := python
+ADDITIONAL_PROPS := \
+	generateSourceCodeOnly=false\
+	hideGenerationTimestamp=true\
+	library=urllib3\
+	packageName=osparc\
+	packageUrl=https://github.com/$(GIT_USER_ID)/${GIT_CLIENT_REPO_ID}.git\
```
[the remark this answers is not in the document]
Comment (bisgaard-itis) [reply scripts/common.Makefile:33]: Seems there is. I seem to be able to do this with `git config --get remote.origin.url`

**Comment by bisgaard-itis** [comment]
@pcrespov I had to fix the ubuntu version for the ci tests to 20.04 in order to be able to run the tests with python 3.6. Let me know if this is not OK for you. Then I will correct in the next PR

**Pull request merged**